src/main/java/org/primefaces/component/chips/ChipsRenderer.java

```
@@ -54,6 +54,10 @@ public void decode(FacesContext context, UIComponent component) {
             submittedValues = LangUtils.concat(submittedValues, new String[]{inputValue});
         }
 
+        if (submittedValues.length > chips.getMax()) {
+            return;
+        }
```

Should we throw an exception instead?

---

Good point, considered that as well, but decided to silently ignore. There are both (or even more) variants in already existing code. And we don't have any guidelines on that, do we?

---

Yeah I noticed it was not consistent either.  In this case its an unexpected scenario so I thought maybe a FacesException would be better.

---

I just remember that we switched from throw a exception to silent ignore it for another case. 
But can't remember, maybe we should work on a guideline.

---

@tandraschko Should we change it to throw a `FacesException `instead of silently ignoring it?

---

I would leave it for now. But it think we should create a issue for 7.1 to align all validations.

---

Agreed. Thanks guys.